src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/NamespaceCache.java

```diff
@@ -72,13 +86,20 @@ protected Uni<CollectionSettings> getCollectionProperties(String collectionName)
     }
   }
 
+  @VisibleForTesting
```

Comment:
Is this needed? I thought it was for case where method has "too much" visibility for purposes of testing?
(as per https://stackoverflow.com/questions/33267820/how-to-use-visiblefortesting-for-pure-junit-tests)

Reply:
Good catch, was trying something, will remove it.